handbook/engineering/tuf.md

```diff
@@ -0,0 +1,138 @@
+
+There are two roles required to complete these steps, the "Root" role who holds the root keys, and the "Releaser" role, who is gaining access to push updates. 
+
+1. The Releaser creates a new local directory to store the TUF repo. The Releaser creates a sub-directory called `repository`.
+
+2. The Realeaser pulls down the contents of the TUF repo into the `repository` sub-directory. 
+
+3. From the root of their TUF directory, the Releaser run `tuf gen-key targets`. This will create a `keys` sub-directory and `staged` sub-directory. 
+
+4. The Releaser copies the `keys` directory to a USB drive, and deletes the `keys` directory from their local hard drive. 
+
+5. The Releaser sends the `staged/root.json` to the Root role for signing. Note this file is safe to share and is publicly available. 
+
+6. The Root role receives the `staged/root.json` file and copies it to a USB drive. 
+
+7. The Root role boots into the secure Ubuntu boot drive created during TUF repo creation. 
+
+8. The Root role connects the USB drive containing the `staged/root.json` file for signing. 
+
+9. The Root role connects the USB drive containing the root keys. 
+
+10. The Root role copies the `staged/root.json` onto the root keys USB at `staged/root.json`. 
+
+11. The root keys USB contains the `tuf` binary. Run `./tuf sign root.json` to sign the staged root metadata. 
+
+12. The Root role copies the signed `staged/root.json` back to the original USB drive they copied it from. 
+
+13. The Root role turns off the Ubuntu boot drive and accesses an online computer. 
+
+14. The Root role connects the USB drive containing the signed `staged/root.json` file and copies it to their local hard drive's TUF location in the same `staged/root.json`. 
+
+15. From the root of their local TUF repo, the Root role runs `tuf commit` to commit the staged root metadata to the `repository` directory. 
+
+16. The Root role pushes the updated contents of the `repository` directory to the remote TUF server. 
+
+17. The Releser role can now run `tuf sign` to sign agent updates using their offline Targets key.
+
+## Rotate the root keys 
+
+The root keys expire every year and must be manually rotated at least 30 days prior to expiration. 
+
+1. The root keys are retrieved from their secure location. 
+
+2. The offline Ubuntu bootable USB drive is turned on. 
+
+3. The root keys USB drive is connected to the Ubuntu bootable instance. 
+
+4. Add three new root keys using the steps documented in creating a new TUF repo. 
+
+5. Run `tuf sign root.json` to sign the newly added root keys with an existing root key. 
+
+6. Run `tuf commit` to commit the staged metadata with new root keys. 
+
+7. Using one of the new root keys, run `tuf revoke-key <role> <id>`. Run this command for each of the old, expiring root keys. 
+
+8. Using one of the new root keys, run `tuf sign root.json` to sign the root metadata removing the old root keys. 
```

Comment:
```suggestion
8. Using each of the new root keys, run `tuf sign root.json` to sign the root metadata removing the old root keys and adding the new keys so that the new root.json is signed by all root keys.
```